### PR TITLE
Improvement/zenko 4784 logging

### PR DESCRIPTION
**Pull request by ghost**
Objectives:
- Better logging of ctst tests (verbose when test fails, otherwise, up to the config - default is silent)
- Better generation of random string to avoid conflicts during the 1000s of policy tests

Example in 
https://github.com/scality/Zenko/actions/runs/8706259439/job/23878514676?pr=2059

Log examples in case of scenario failure:
```log
2024-04-16T07:58:24.4991884Z {"name":"CTST","reqId":136,"api":"CreateBucketCommand","input":{"Bucket":"zenko-ctstbucketfortestxpsktkbm9qng8w4qs2jr"},"response":"{\"$metadata\":{\"httpStatusCode\":200,\"requestId\":\"9bc40a46b348820ecf83\",\"extendedRequestId\":\"9bc40a46b348820ecf83\",\"attempts\":1,\"totalRetryDelay\":0},\"Location\":\"/zenko-ctstbucketfortestxpsktkbm9qng8w4qs2jr\"}","accessKey":"YGJHKGW48PSASBVGQX1B","secretKey":"ohhDgx0Hp+m0mDdPROhpwOFY2eh812te2Ew9gej5","isSessionToken":false,"endpoint":"http://s3.zenko.local:80","time":1713254221140,"req_id":"0f196129114992208559","level":"debug","message":"Command results","hostname":"end2end-ctst-tests","pid":2250}
2024-04-16T07:58:24.5001208Z {"name":"CTST","reqId":137,"api":"GetBucketNotificationConfigurationCommand","input":{"Bucket":"zenko-ctstbucketfortestxpsktkbm9qng8w4qs2jr"},"response":"{\"$metadata\":{\"httpStatusCode\":200,\"requestId\":\"895c6a35a0499e33058b\",\"extendedRequestId\":\"895c6a35a0499e33058b\",\"attempts\":1,\"totalRetryDelay\":0}}","accessKey":"YGJHKGW48PSASBVGQX1B","secretKey":"ohhDgx0Hp+m0mDdPROhpwOFY2eh812te2Ew9gej5","isSessionToken":false,"endpoint":"http://s3.zenko.local:80","time":1713254221152,"req_id":"0f196129114992208559","level":"debug","message":"Command results","hostname":"end2end-ctst-tests","pid":2250}
2024-04-16T07:58:24.5013509Z {"name":"CTST","reqId":138,"api":"PutBucketNotificationConfigurationCommand","input":{"Bucket":"zenko-ctstbucketfortestxpsktkbm9qng8w4qs2jr","NotificationConfiguration":{"QueueConfigurations":[{"QueueArn":"arn:scality:bucketnotif:::destination1","Events":["s3:ObjectCreated:Put"]}]}},"response":"{\"$metadata\":{\"httpStatusCode\":200,\"requestId\":\"0b532ba75a4e80d0e3a5\",\"extendedRequestId\":\"0b532ba75a4e80d0e3a5\",\"attempts\":1,\"totalRetryDelay\":0}}","accessKey":"YGJHKGW48PSASBVGQX1B","secretKey":"ohhDgx0Hp+m0mDdPROhpwOFY2eh812te2Ew9gej5","isSessionToken":false,"endpoint":"http://s3.zenko.local:80","time":1713254221171,"req_id":"0f196129114992208559","level":"debug","message":"Command results","hostname":"end2end-ctst-tests","pid":2250}
2024-04-16T07:58:24.5026669Z {"name":"CTST","reqId":139,"api":"GetBucketNotificationConfigurationCommand","input":{"Bucket":"zenko-ctstbucketfortestxpsktkbm9qng8w4qs2jr"},"response":"{\"$metadata\":{\"httpStatusCode\":200,\"requestId\":\"e3252d784446af9c8453\",\"extendedRequestId\":\"e3252d784446af9c8453\",\"attempts\":1,\"totalRetryDelay\":0},\"QueueConfigurations\":[{\"Id\":\"N2JjMTgxM2YtMGZiZS00NTAyLThmODQtN2JjMzk5YTMwYTZl\",\"QueueArn\":\"arn:scality:bucketnotif:::destination1\",\"Events\":[\"s3:ObjectCreated:Put\"]}]}","accessKey":"YGJHKGW48PSASBVGQX1B","secretKey":"ohhDgx0Hp+m0mDdPROhpwOFY2eh812te2Ew9gej5","isSessionToken":false,"endpoint":"http://s3.zenko.local:80","time":1713254231194,"req_id":"0f196129114992208559","level":"debug","message":"Command results","hostname":"end2end-ctst-tests","pid":2250}
2024-04-16T07:58:24.5035663Z {"name":"CTST","reqId":140,"api":"PutBucketNotificationConfigurationCommand","input":{"Bucket":"zenko-ctstbucketfortestxpsktkbm9qng8w4qs2jr","NotificationConfiguration":{"$metadata":{"httpStatusCode":200,"requestId":"e3252d784446af9c8453","extendedRequestId":"e3252d784446af9c8453","attempts":1,"totalRetryDelay":0},"QueueConfigurations":[{"Id":"N2JjMTgxM2YtMGZiZS00NTAyLThmODQtN2JjMzk5YTMwYTZl","QueueArn":"arn:scality:bucketnotif:::destination1","Events":["s3:ObjectCreated:Put"]},{"QueueArn":"arn:scality:bucketnotif:::destination2","Events":["s3:ObjectCreated:Put"]}]}},"response":"{\"$metadata\":{\"httpStatusCode\":200,\"requestId\":\"9a28e748ef44a1c8ce34\",\"extendedRequestId\":\"9a28e748ef44a1c8ce34\",\"attempts\":1,\"totalRetryDelay\":0}}","accessKey":"YGJHKGW48PSASBVGQX1B","secretKey":"ohhDgx0Hp+m0mDdPROhpwOFY2eh812te2Ew9gej5","isSessionToken":false,"endpoint":"http://s3.zenko.local:80","time":1713254231211,"req_id":"0f196129114992208559","level":"debug","message":"Command results","hostname":"end2end-ctst-tests","pid":2250}
2024-04-16T07:58:24.5044872Z {"name":"CTST","reqId":141,"api":"PutObjectCommand","input":{"Bucket":"zenko-ctstbucketfortestxpsktkbm9qng8w4qs2jr","Key":"object-xi9f4o5dgsp47sfld3u5"},"response":"{\"$metadata\":{\"httpStatusCode\":200,\"requestId\":\"c4fe545a6c4db1a01232\",\"extendedRequestId\":\"c4fe545a6c4db1a01232\",\"attempts\":1,\"totalRetryDelay\":0},\"ETag\":\"\\\"d41d8cd98f00b204e9800998ecf8427e\\\"\"}","accessKey":"YGJHKGW48PSASBVGQX1B","secretKey":"ohhDgx0Hp+m0mDdPROhpwOFY2eh812te2Ew9gej5","isSessionToken":false,"endpoint":"http://s3.zenko.local:80","time":1713254241236,"req_id":"0f196129114992208559","level":"debug","message":"Command results","hostname":"end2end-ctst-tests","pid":2250}
2024-04-16T07:58:24.5055324Z {"name":"CTST","reqId":142,"api":"ListObjectsCommand","input":{"Bucket":"zenko-ctstbucketfortestxpsktkbm9qng8w4qs2jr"},"response":"{\"$metadata\":{\"httpStatusCode\":200,\"requestId\":\"2b7658de3740b5d03a16\",\"extendedRequestId\":\"2b7658de3740b5d03a16\",\"attempts\":1,\"totalRetryDelay\":0},\"Contents\":[{\"Key\":\"object-xi9f4o5dgsp47sfld3u5\",\"LastModified\":\"2024-04-16T07:57:21.233Z\",\"ETag\":\"\\\"d41d8cd98f00b204e9800998ecf8427e\\\"\",\"Size\":0,\"StorageClass\":\"STANDARD\",\"Owner\":{\"DisplayName\":\"zenko-ctst\",\"ID\":\"2e2a37d044c8fbd355be6e177d87f5d85e70bdbca52f890254be6557b632a587\"}}],\"IsTruncated\":false,\"Marker\":\"\",\"MaxKeys\":1000,\"Name\":\"zenko-ctstbucketfortestxpsktkbm9qng8w4qs2jr\",\"Prefix\":\"\"}","accessKey":"YGJHKGW48PSASBVGQX1B","secretKey":"ohhDgx0Hp+m0mDdPROhpwOFY2eh812te2Ew9gej5","isSessionToken":false,"endpoint":"http://s3.zenko.local:80","time":1713254244384,"req_id":"0f196129114992208559","level":"debug","message":"Command results","hostname":"end2end-ctst-tests","pid":2250}
2024-04-16T07:58:24.5064687Z {"name":"CTST","reqId":143,"api":"DeleteObjectCommand","input":{"Bucket":"zenko-ctstbucketfortestxpsktkbm9qng8w4qs2jr","Key":"object-xi9f4o5dgsp47sfld3u5"},"response":"{\"$metadata\":{\"httpStatusCode\":204,\"requestId\":\"8288dcc7c64c9f27128e\",\"extendedRequestId\":\"8288dcc7c64c9f27128e\",\"attempts\":1,\"totalRetryDelay\":0}}","accessKey":"YGJHKGW48PSASBVGQX1B","secretKey":"ohhDgx0Hp+m0mDdPROhpwOFY2eh812te2Ew9gej5","isSessionToken":false,"endpoint":"http://s3.zenko.local:80","time":1713254244423,"req_id":"0f196129114992208559","level":"debug","message":"Command results","hostname":"end2end-ctst-tests","pid":2250}
2024-04-16T07:58:24.5074855Z {"name":"CTST","reqId":144,"api":"DeleteBucketCommand","input":{"Bucket":"zenko-ctstbucketfortestxpsktkbm9qng8w4qs2jr"},"response":"{\"$metadata\":{\"httpStatusCode\":204,\"requestId\":\"70a237e9d84299487852\",\"extendedRequestId\":\"70a237e9d84299487852\",\"attempts\":1,\"totalRetryDelay\":0}}","accessKey":"YGJHKGW48PSASBVGQX1B","secretKey":"ohhDgx0Hp+m0mDdPROhpwOFY2eh812te2Ew9gej5","isSessionToken":false,"endpoint":"http://s3.zenko.local:80","time":1713254244473,"req_id":"0f196129114992208559","level":"debug","message":"Command results","hostname":"end2end-ctst-tests","pid":2250}
2024-04-16T07:58:24.5079129Z {"name":"CTST","scenario":"Recieve notification in multiple destinations","status":"FAILED","time":1713254304497,"req_id":"0f196129114992208559","level":"error","message":"Scenario failed.","hostname":"end2end-ctst-tests","pid":2250}
```

### CODE REVIEW

**Review by BourgoisMickael — APPROVED** [review]

File: tests/ctst/package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "@typescript-eslint/eslint-plugin": "^5.45.0",
     "@typescript-eslint/parser": "^5.45.0",
     "babel-jest": "^29.3.1",
-    "cli-testing": "github:scality/cli-testing.git#0.3.1",
+    "cli-testing": "github:scality/cli-testing.git#dd6bb9ca6dccb3fe38cf6c156dcbc208f22c7ca8",
```
Comment (KillianG) [inline tests/ctst/package.json]: Adding this as a reminder to change tag to next release of clitesting otherwise lgtm

Comment (ghost) [reply tests/ctst/package.json]: Yes, done post release of ctst 0.4.1

File: .github/workflows/end2end.yaml [hunk 2]
```diff
@@ -32,7 +32,7 @@ env:
   BACKBEAT_LCC_CRON_RULE: ""
   # e2e-env
   E2E_IMAGE_NAME: registry.scality.com/zenko/zenko-e2e
-  E2E_CTST_IMAGE_NAME: registry.scality.com/zenko/zenko-e2e-ctst
+  E2E_CTST_IMAGE_NAME: ghcr.io/scality/zenko/zenko-e2e-ctst
```
Comment (francoisferrand) [inline .github/workflows/end2end.yaml]: not yet, keep pushing to old registry for now
(migration will be done separately)

**Review by francoisferrand — APPROVED** [review]

**Pull request merged**